android/src/main/java/com/reactnativechangeicon/ChangeIconModule.java

```diff
@@ -100,6 +99,8 @@ private void completeIconChange() {
         final Activity activity = getCurrentActivity();
         if (activity == null)
             return;
+        
+        classesToKill.removeIf(cls -> cls.equals(componentClass));
```

Comment:
can you change this to `classesToKill.remove(componentClass)`
i dont think `removeIf` is required

Reply:
Sure